helm-swoop.el

```diff
@@ -282,13 +282,11 @@
   (recenter (/ (window-height) 2)))
 
 (defsubst helm-swoop--delete-overlay ($identity &optional $beg $end)
-  (or $beg (setq $beg (point-min)))
```

Comment:
Use `(cl-defun some-func (&optional (arg default-val)))` to specify default value for arguments